dist/infragram.js

```diff
@@ -532,6 +532,45 @@ function _slicedToArray(arr, i) { if (Array.isArray(arr)) { return arr; } else i
     // This file was adapted from infragram-js:
     // http://github.com/p-v-o-s/infragram-js.
     module.exports = function File(options, processor) {
+      var _getMouseEvent = function getMouseEvent() {
```

Comment:
Just noting that all this looks fine - these changes are from https://github.com/publiclab/infragram/pull/324 and correctly match the `main` branch. It's possible that https://github.com/publiclab/infragram/pull/324 didn't run `grunt build` and so this is the first time they're getting copied (compiled) into the `dist` files from the `src` files? But I don't see an issue.